index.js

```diff
@@ -1,11 +1,22 @@
 require('dotenv').config();
-
-const {Client} = require('discord.js');
+const fs = require("fs");
+const {Client, Collection} = require('discord.js');
 const Bot = require('./src/Bot.js');
 const XkcdSource = require('./src/Source/XkcdSource.js');
 
 const client = new Client();
+client.commands = new Collection();
+
+const commandFiles = fs.readdirSync('./src/commands').filter(file => file.endsWith('.js'));
+
+for (const file of commandFiles) {
+  const command = require(`./src/commands/${file}`)
+
+  client.commands.set(command.name, command)
+}
+
 const bot = new Bot();
+module.exports = bot;
```

Comment:
I'm not a big fan of making the bot instance globally available tbh.

Reply:
I'm in agreement. Did you see my other comment about a potential way to structure this instead?

Reply:
Just read up on it. Sounds good :) I do like the idea of separating the commands into specific handlers.. Might improve extensibility and testability.